halo2_proofs/src/dev/gates.rs

```diff
@@ -158,7 +159,7 @@ impl CircuitGates {
                                     .into_iter()
                                     .collect()
                             },
-                            &|_, column, rotation| {
+                            &|_, column, rotation, _| {
```

Comment:
Ditto.

Reply:
Both updated in [`c87ba7a`](https://github.com/zcash/halo2/pull/593/commits/c87ba7a4a403d8c6f201134fece8f0af5375f8d4#diff-801c054c192c4d61fb5811b798f45a71d3dde952c5716ece6787f79c32d4f27d).